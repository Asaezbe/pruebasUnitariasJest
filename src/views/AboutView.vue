<template>
    <div class="Contact">
  
    
      <h1>Somos gente genial.<br>De momento construimos un sitio mejor.</h1>
    </div>
    
  </template>


<script>
  import AboutView from './AboutView.vue';

  // @ is an alias to /src
  
  export default {
    name: 'AboutView',
    components: {
      AboutView
  }
  }
  </script>