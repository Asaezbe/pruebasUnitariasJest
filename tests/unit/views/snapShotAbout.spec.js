///Foto test estructura html componente About

import { mount } from '@vue/test-utils'
import AboutView from '@/views/AboutView.vue'

describe('AboutView Snapshot', () => {
    test('deberÃ­a hacer match con el snapshot actual', () => {
        // Montar el componente
        const wrapper = mount(AboutView)

        // Crear el snapshot de la estructura HTML
        expect(wrapper.html()).toMatchSnapshot()
    })
})