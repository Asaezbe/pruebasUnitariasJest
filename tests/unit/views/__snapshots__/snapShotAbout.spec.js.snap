// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutView Snapshot deber√≠a hacer match con el snapshot actual 1`] = `
<div class="Contact">
  <h1>Somos gente genial.<br>De momento construimos un sitio mejor.</h1>
</div>
`;
